### ISSUE TITLE
Coding convention

### ISSUE THREAD

**Issue by nicelhc13** [ut-parla/parla-experimental#2]
It would be better if we have a coding convention. I enjoy using Google's C++ coding convention: https://google.github.io/styleguide/

**Comment by dialecticDolt** [comment]
Yeah I agree we we should use something. Let me look through this. 

Some comments:

We should keep our C++ comments compatible to doxygen. 

Unless we want to split runtime.hpp we'll need the forward declarations. 

I've tend to really like underscores for methods, functions, and variables as you can see 😂. I find them the most readable.
